### DISCUSSION TITLE
Cloning Repository

### DISCUSSION THREAD

**Discussion by Sukritish1299** [community/community#31934] [Other Feature Feedback, Questions, & Ideas] [answered]
I am trying to clone a repository. It shows 100% but only one single file out of 4 is not cloning. I keep trying from the beginning but it's the same. Please help.

**Comment by airtower-luna** [comment]
Please post the exact command you're using to clone and its full output, otherwise it's impossible to guess what might be the problem.

**Comment by ihnatenkoo** [comment]
Hello, try this comand in repository
`git fetch -p && git pull`

**Comment by Sukritish1299** [comment]
I am writing this
git clone link.git
Cloning into 'link'...
remote: Enumerating objects: 2682, done.
remote: Counting objects: 100% (1511/1511), done.
remote: Compressing objects: 100% (729/729), done.
remote: Total 2682 (delta 1026), reused 991 (delta 742), pack-reused 1171
Receiving objects: 100% (2682/2682), 876.46 KiB | 796.00 KiB/s, done.
Resolving deltas: 100% (1625/1625), done.

It says finished 100% but I can't see the folder name data in here which is on the repository.

**Comment by Sukritish1299** [comment]
It is a folder named 'data' and contains 5 JSON Source Files.

**Comment by airtower-luna** [comment, marked as answer]
Are you on the same branch as the one where you see those JSON files? You can check the branch and general repository status with `git status`.
